clang/include/clang/Driver/Action.h

```diff
@@ -793,18 +793,34 @@ class FileTableTformJobAction : public JobAction {
   // <To> from another file table passed as input to this action.
   void addReplaceColumnTform(StringRef From, StringRef To);
 
+  // Replaces a cell in this table with column title <ColumnName> and row <Row>
+  // with the file name passed as input to this action.
+  void addReplaceCellTform(StringRef ColumnName, int Row);
+
   // Renames a column with title <From> in this table with a column with title
   // <To> passed as input to this action.
   void addRenameColumnTform(StringRef From, StringRef To);
 
+  // Specifies that, instead of generating a new table, the transformation
+  // should copy the file in the only remaining row at <ColumnName> into the
+  // output file.
+  void setCopySingleFileColumn(StringRef ColumnName);
```

Comment:
To make this tform more generic, please make the coordinate 2d, so that in can be applied to arbitrary file table with many rows.

Reply:
Good idea. It does potentially hide errors if, for some reason, a change would make tables in the CUDA patch with multiple rows, but I appreciate the generality of this solution and such a bug would probably break in many other places anyway.

I have made the suggested change.